CHANGELOG.md

```diff
@@ -3,10 +3,12 @@
 ## Features
 
 - Add environment variable `BAT_PAGING`, see #2629 (@einfachIrgendwer0815)
+- Add support for `LESSOPEN` and `LESSCLOSE`. See #1597, #1739, and #2444 (@Anomalocaridid)
 
 ## Bugfixes
 
 - Fix `more` not being found on Windows when provided via `BAT_PAGER`, see #2570, #2580, and #2651 (@mataha)
+- Fix `less` preprocessing with `LESSOPEN` again and causing a test to intermittently fail, and incorrect `LESSOPEN` behavior see #2444, #2602, and #2662 (@Anomalocaridid)
```

Comment:
This was was never present in a release version of bat, right? In that case I don't think we should have this as a bugfix. But feel free to list the PRs in the "Features" item.

Reply:
I moved the PRs to the "Features" item. How is it now?